packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -305,7 +286,23 @@ export class GarbageCollector implements IGarbageCollector {
      * throughout its lifetime.
      */
     private readonly gcEnabled: boolean;
+    /**
+     * Tracks if sweep phase is enabled for this document. This is specified during document creation and doesn't change
+     * throughout its lifetime.
+     */
+    private readonly sweepEnabled: boolean;
```

Comment:
I'd like to not name this `sweepEnabled` it feels like if this is true, sweep should be running, maybe we should just be storing `sweepAllowed` based on the runtime option.

Reply:
`sweepEnabled` does mean if its true, sweep will be running. Currently, we have another layer (feature flags) to enable sweep because we want to stage it such that we can test it, run sweep for a subset of audiences, etc. Basically, build confidence that sweep works as expected. Once we remove the feature flag, this is the source of truth for whether sweep runs or not.

If you mean just renaming the class member to `sweepAllowed` for now I can do that. But the property that we write in the metadata blob should be called `sweepEnabled` because we can't change that later.